Library/Homebrew/dev-cmd/release-notes.rb

```diff
@@ -55,25 +56,9 @@ def release_notes
       odie "Ref #{ref} does not exist!"
     end
 
-    output = Utils.popen_read(
-      "git", "-C", HOMEBREW_REPOSITORY, "log", "--pretty=format:'%s >> - %b%n'", "#{previous_tag}..#{end_ref}"
-    ).lines.grep(/Merge pull request/)
-
-    output.map! do |s|
-      s.gsub(%r{.*Merge pull request #(\d+) from ([^/]+)/[^>]*(>>)*},
-             "https://github.com/Homebrew/brew/pull/\\1 (@\\2)")
-    end
-    if args.markdown?
-      output.map! do |s|
-        /(.*\d)+ \(@(.+)\) - (.*)/ =~ s
-        "- [#{Regexp.last_match(3)}](#{Regexp.last_match(1)}) (@#{Regexp.last_match(2)})"
-      end
-    end
+    release_notes = ReleaseNotes.generate_release_notes previous_tag, end_ref, markdown: T.must(args.markdown?)
```

Comment:
```suggestion
    release_notes = ReleaseNotes.generate_release_notes previous_tag, end_ref, markdown: args.markdown?
```

Reply:
Why is this way preferred? The way I see it, `generate_release_notes` should only accept `T::Boolean` for the `markdown` parameter.

It's up to whatever is calling the method to ensure that it's passing a boolean. In this case, `T.must` is appropriate because we can be sure that `args.markdown?` is either `true` or `false` because `--markdown` is one of the arg options.

Reply:
> we can be sure that `args.markdown?` is either `true` or `false` because `--markdown` is one of the arg options.

Correct me if I'm wrong, `args.markdown?` returns `nil` if `--markdown` isn't passed, right?

Reply:
Oops, you're right! In that case, I agree with this change.